bootstrap/app.php

```diff
@@ -86,9 +99,12 @@
 |
 */
 
-// $app->register(App\Providers\AuthServiceProvider::class);
- $app->register(\Domains\Links\LinksServiceProvider::class);
- $app->register(\Domains\Tags\TagsServiceProvider::class);
+//$app->register(App\Providers\AuthServiceProvider::class);
+$app->register(\Illuminate\Notifications\NotificationServiceProvider::class);
+$app->register(\Domains\Links\LinksServiceProvider::class);
+$app->register(\Domains\Tags\TagsServiceProvider::class);
+$app->register(\Domains\Accounts\AccountsServiceProvider::class);
+$app->register(Illuminate\Mail\MailServiceProvider::class);
```

Comment:
Nit: we're missing a `\` before the Illuminate string.

Also, can we keep these registers ordered alphabetically by their FQN? :smile:

Reply:
sure